### PR TITLE
added clothes shop

### PR DESCRIPTION
**Pull request by TooDeepForSleep**

### CODE REVIEW

**Comment by maj113** [comment]
Could use some improvements, I'll address those later, we will definitely need more clothes or at least "branded" clothes

File: Activities/clothesShop.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+from Utilities.loader import  load_json
+from Player.Playerstats import update_player_stats
+
+def load_clothes_json():
+    clothes_shop = load_json("./Activities/clothesShop.json")
+    clothes_in_shop = clothes_shop.get("clothes", {})
+    return clothes_in_shop
+
+def clothes_Shop():
+    """Go to clothes shop"""
```
Comment (maj113) [inline Activities/clothesShop.py]: """Go to clothes shop""" sounds a bit weird

We could do "Go shopping for clothes "

**Comment by TooDeepForSleep** [comment]
Shut yo ass up

**Comment by maj113** [comment]
LGTM

**Pull request merged**